app/views/provider_interface/provider_relationship_permissions_setup/organisations.html.erb

```diff
@@ -0,0 +1,35 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">
+      Set up permissions for your organisation
+    </h1>
+  </div>
+</div>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds govuk-body">
+    <p>You can control who has access to sensitive information, and who can make decisions about applications.</p>
+    <p>We’ll ask you to set up the permissions between you and the organisations that ratify your courses.</p>
+    <p>You’ll then be able to choose permissions for any individual users you add to your organisation.</p>
+    <p>You can change these settings at any time.</p>
+
+    <h2 class="govuk-heading-m">The organisations you’ll need to set up</h2>
+
+    <% @grouped_provider_names_from_relationships.each do |training_provider_name, ratifying_provider_names| %>
```

Comment:
On the prototype when there is only one provider, we don't see the provider's name. Is this correct @adamsilver ?

Reply:
We can revisit this in another PR